lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -76,10 +78,18 @@ def self.base_model
 
     def self.model
       # Set a class-instance variable to get the appropriate model
-      @model ||= Object.const_get(/MiqAeService(.+)$/.match(name)[1].gsub(/_/, '::'))
+      @model ||= /MiqAeService(.+)$/.match(name)[1].gsub(/_/, '::').constantize
```

Comment:
@jrafanie suggestions very welcome :smile:... I tried and failed to come up with a less-awful alternative, but that doesn't mean one doesn't exist.